packages/@react-aria/dnd/docs/useDroppableCollection.mdx

```diff
@@ -0,0 +1,1080 @@
+
+In the below example, the drop target only supports dropping PNG images. If a PNG is dragged over the target, it will be highlighted and the operating system displays a copy cursor. If another type is dragged over the target, then there is no visual feedback, indicating that a drop is not accepted there. If the user holds a modifier key such as <Keyboard>Control</Keyboard> while dragging over the drop target in order to change the drop operation, then the drop target does not accept the drop.
+
+```tsx example
+function Example() {
+  ///- begin collapse -///
+  let list = useListData({});
+  let onRootDrop = async e => {
+    let items = await Promise.all(
+      e.items
+        .map(async item => ({
+          id: Math.random(),
+          url: URL.createObjectURL(await item.getFile()),
+          name: item.name
+        }))
+    );
+
+    list.prepend(...items);
+  };
+  ///- end collapse -///
+  // ...
+
+  return (
+    <ListBox
+      aria-label="Images"
+      items={list.items}
+      /*- begin highlight -*/
+      getDropOperation={() => 'copy'}
+      /*- end highlight -*/
+      acceptedDragTypes={['image/png']}
+      onRootDrop={onRootDrop}>
+      {item => (
+        <Item>
+          <div className="image-item">
+            <img src={item.url} />
+            <span>{item.name}</span>
+          </div>
+        </Item>
+      )}
+    </ListBox>
+  );
+}
+```
+
+### Drop events
+
+Drop events such as `onInsert`, `onItemDrop`, etc. also include the `dropOperation`. This can be used to perform different actions accordingly, for example, when communicating with a backend API.
+
+```tsx
+let onItemDrop = async (e) => {
+  let data = JSON.parse(await e.items[0].getText('my-app-file'));
+  /*- begin highlight -*/
+  switch (e.dropOperation) {
+    case 'move':
+      MyAppFileService.move(data.filePath, props.filePath);
+      break;
+    case 'copy':
+      MyAppFileService.copy(data.filePath, props.filePath);
+      break;
+    case 'link':
+      MyAppFileService.link(data.filePath, props.filePath);
+      break;
+  }
+  /*- end highlight -*/
+};
+```
+
+## Low level API
+
+The above examples have used high level events such as `onInsert`, `onItemDrop`, and `onReorder`, along with props such as `acceptedDragTypes`. Based on these props, `useDroppableCollection` automatically determines whether drag data is accepted and where (e.g. on items, between items, etc.). It also automatically filters the dropped items based on their types. For more complex scenarios, the lower level `getDropOperation` and `onDrop` functions can be used instead.
+
+To programmatically determine whether a drop is accepted based on the dragged types, target key, and drop position, implement the `getDropOperation` function. `acceptedDragTypes` and `shouldAcceptItemDrop` may also be used in combination with `getDropOperation` as a pre-filter. See the section on [drop operations](#drop-operations) above for more details.
```

Comment:
At the moment `shouldAcceptItemDrop` is only used as a pre-filter when `onItemDrop` is provided